.devcontainer/Dockerfile

```diff
@@ -0,0 +1,109 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
```

Comment:
Also, just caught this. Can we MIT license this and copyright it to Microsoft under the current Crossplane license and governance? I suspect we will at least have to use the Apache 2.0 license the rest of the project uses, and attribute copyright to the Crossplane authors.

Reply:
Good point. Let me adhere to the existing license.